### DISCUSSION TITLE
2005/08/11/insects

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#45] [Announcements]
# 2005/08/11/insects

My favorite TV Channel is the Shanghai Documentary Channel, because I can always enjoy the graceful creatures on it.  I’ll intrude one kind of creatures today, insect.

https://blog.qizhen.xyz/2005/08/11/insects

**Comment by ruanqizhen** [comment]
JING
2005年08月13日 18:52 编辑
我是学医的,我一个同学双学位修的是计算机,他跟我说,医学跟计算机合在一起,会害死人的!!看来站长是强人中的强人啊!!

回复
OLIVIA
2005年08月12日 14:49 编辑
好专业~~~

回复
